bindings/julia/Displaz.jl

```diff
@@ -50,15 +49,54 @@ function write_ply_points(fileName, nvertices, fields)
             for i = 1:size(value,2)
                 propName = ply_property_name(semantic, i)
                 write(fid, "property $typeName $propName\n")
-            end
+        	end
         end
         write(fid, "end_header\n")
         for (_,value) in convertedFields
-            write(fid, value')
+			write(fid, value')
         end
     end
 end
 
+function write_ply_lines(fileName, position, color)
+	vertexValid = ~all(isnan(position),2)
```

Comment:
On second thoughts, let's kill off the NaNs now.  I suggest that we simply pass the equivalent of lineStarts as a positional argument in the interface - let's call it `linebreak` (or some better name?).  In the plot function it could be a keyword argument, with a default of `[1]`.  Example usage via plot:

```julia
# Plot two separate line segments, with the index of the first starting at 1,
# and the index of the second starting at 11.  Internally this should result
# in the ranges `1:10` and `11:20` for the two line segments.
Displaz.plot(randn(20,3), markershape='-', linebreak=[1,11])
```